cran-comments.md

```diff
@@ -1,11 +1,17 @@
 ## Release summary
 This is a minor release.
 
+Adressing the note in R CMD CHECK:
```

Comment:
can you move this explanation under the R CMD check results section

Reply:
done